package.json

```diff
@@ -14,9 +14,11 @@
     "dev": "webpack-dev-server"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
     "@babel/plugin-transform-runtime": "^7.2.0",
     "@babel/preset-env": "^7.3.1",
     "@babel/runtime-corejs2": "^7.3.1",
+    "babel-core": "^7.0.0-0",
```

Comment:
do we need both babel cores?

Reply:
I'm not sure, I copied this over from concert-ads. I'll test with one version and see if there are still errors.

Reply:
Jenine, once this tested, lets ship it!

I can merge, or you can :)

Reply:
Hi Brian! OK, so I dug into the babel cores and why both are used in some cases. It looks like since Babel 7, Babel switched to scoped packages and `@babel/core` vs `babel-core`. The IE issue is fixed with just `@babel/core`! And more importantly, it looks like no packages require (a bridge package solution for backwards compatibility)[https://github.com/babel/babel-bridge]. Shipping!!